test/apiv2/rest_api/__init__.py

```diff
@@ -0,0 +1,132 @@
+import configparser
+import json
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+
+
+class Podman(object):
+    """
+    Instances hold the configuration and setup for running podman commands
+    """
+
+    def __init__(self):
+        """Initialize a Podman instance with global options"""
+        binary = os.getenv("PODMAN", "bin/podman")
+        self.cmd = [binary, "--storage-driver=vfs"]
+
+        cgroupfs = os.getenv("CGROUP_MANAGER", "cgroupfs")
+        self.cmd.append(f"--cgroup-manager={cgroupfs}")
+
+        if os.getenv("DEBUG"):
+            self.cmd.append(["--log-level=debug"])
```

Comment:
This is the culprit. `$DEBUG` is set in the CI environment, and this statement converts `cmd` into a list of lists. Fix: remove the brackets. (And, if I may make a recommendation, use a less ambiguous envariable such as `$PODMAN_REST_API_DEBUG`)

Reply:
Ahwww dangit :confounded: This is the second time my "brilliant" choice of `$DEBUG` for the shared automation library has broken something.  Before it gets even more out-of-hand, would either you or @edsantiago be willing to [fix the problem at the source?](https://github.com/containers/automation/blob/master/common/lib/defaults.sh#L10).  I'm very much focused on getting F33 VMs working w/ podman CI, then I need to focus on branch-failure notifications.  Help would be much appreciated.

Reply:
The bug is in the code. `$DEBUG` triggered it, but was not the cause. 

I think we should make a constant effort to use descriptive conflict-free envariables, but there is no rush to fix the CI setup.

Reply:
still `$DEBUG` was a poor choice on my part...in a long consistent line of poor name choices :disappointed: Frankly, I simply no longer trust myself to come up with good names.  It's just not in my DNA :cry:

Reply:
@edsantiago Good catch! I copied the code from our other repo's and missed that.